metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -184,16 +202,21 @@ def __repr__(self):
     def _apply(self, data):
         self._data = data
 
-    @limit(1)
+    @Throttle()
     def _update(self):
         try:
             data = self._client.describe_jobs(jobs=[self._id])
-        except self._client.exceptions.ClientException:
-            return
+        except self._client.exceptions.ClientError as err:
+            code = err.response['ResponseMetadata']['HTTPStatusCode']
```

Comment:
Are we sure that all err have this data. Otherwise maybe do something like try: code = err.response[][] except: raise err

Reply:
ClientError is guaranteed to have this data according to their docs.